cs251tk/toolkit/__main__.py

```diff
@@ -123,7 +124,9 @@ def main():
 
         else:
             for student in usernames:
-                print('processing {}'.format(student))
+                logging.debug('\nProcessing {}'.format(student))
```

Comment:
Additionally, I'm 99% positive that the logging system will put them on newlines, so could we remove the \n from these, please?

Reply:
Those were there to space each student apart. If formatting doesn't matter, they can be removed.